Update changelog.txt (#3282)
AzureAD/microsoft-authentication-library-for-dotnet: changelog.txt

4.43.1
==========

> ### Bug fix
> 
> **MSAL uses WebView1 instead of WebView2 for AcquireTokenInteractive with AAD or ADFS authority due to SSO issues**. See [3270](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3270)

Reviewer: ```suggestion
### Bug Fix

**MSAL now uses WebView1 instead of WebView2 for `AcquireTokenInteractive` with AAD or ADFS authority because WebView2 doesn't support SSO**. See [3270](https://github.com/AzureAD/microsoft-authentication-library-for-dotnet/issues/3270)
```
